test/mpi/dtpools/src/dtpools.c

```diff
@@ -116,19 +140,13 @@ int DTP_pool_create_struct(int num_types, MPI_Datatype * basic_types, int *basic
 {
     int i, err = DTP_SUCCESS;
     int num_objs;
-    int env_num_objs = INT_MAX;
-    char *env_num_objs_str = NULL;
+    int min_obj_idx, max_obj_idx;
     struct DTP_obj_array_s *obj_array = NULL;
 
     DTPI_OBJ_ALLOC_OR_FAIL(*dtp, sizeof(**dtp));
 
-    /* get number of objects from environment */
-    if ((env_num_objs_str = getenv("DTP_NUM_OBJS"))) {
-        env_num_objs = atoi(env_num_objs_str);
-    }
-
-    num_objs = (env_num_objs < DTPI_OBJ_LAYOUT__STRUCT_NUM && env_num_objs > 0) ?
-        env_num_objs : DTPI_OBJ_LAYOUT__STRUCT_NUM;
+    get_obj_id_range(DTPI_OBJ_LAYOUT__STRUCT_NUM - 1, &min_obj_idx, &max_obj_idx);
```

Comment:
There is only one struct layout so `DTPI_OBJ_LAYOUT__STRUCT_NUM` is equal to `1`. Wouldn't this pass a default of `0`?

Reply:
Same response as above.